### ISSUE TITLE
cannot find native library LibJavaPcre.so

### ISSUE THREAD

**Issue by eemhu** [teragrep/jpr_01#13]
on jpr_01 3.0.0, `JavaPcre pcre = new JavaPcre(); ` causes
```
java.lang.UnsatisfiedLinkError: Unable to load library 'JavaPcre':
libJavaPcre.so: cannot open shared object file: No such file or directory
libJavaPcre.so: cannot open shared object file: No such file or directory
Native library (linux-x86-64/libJavaPcre.so) not found in resource path (...)
	at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:307)
	at com.sun.jna.NativeLibrary.getInstance(NativeLibrary.java:467)
	at com.sun.jna.Library$Handler.<init>(Library.java:192)
	at com.sun.jna.Native.load(Native.java:622)
	at com.sun.jna.Native.load(Native.java:596)
	at com.teragrep.jpr_01.JavaPcre$LibJavaPcre.<clinit>(JavaPcre.java:39)
	at sun.misc.Unsafe.ensureClassInitialized(Native Method)
	at sun.reflect.UnsafeFieldAccessorFactory.newFieldAccessor(UnsafeFieldAccessorFactory.java:43)
	at sun.reflect.ReflectionFactory.newFieldAccessor(ReflectionFactory.java:156)
	at java.lang.reflect.Field.acquireFieldAccessor(Field.java:1088)
	at java.lang.reflect.Field.getFieldAccessor(Field.java:1069)
	at java.lang.reflect.Field.get(Field.java:393)
	at com.sun.jna.Native.loadLibraryInstance(Native.java:694)
	at com.sun.jna.Native.getLibraryOptions(Native.java:764)
	at com.sun.jna.Native.getStructureAlignment(Native.java:856)
	at com.sun.jna.Structure.setAlignType(Structure.java:291)
	at com.sun.jna.Structure.<init>(Structure.java:208)
	at com.sun.jna.Structure.<init>(Structure.java:204)
	at com.sun.jna.Structure.<init>(Structure.java:191)
	at com.sun.jna.Structure.<init>(Structure.java:183)
	at com.teragrep.jpr_01.JavaPcre$LibJavaPcre$OptionsStruct.<init>(JavaPcre.java:42)
	at com.teragrep.jpr_01.JavaPcre.<init>(JavaPcre.java:176)
```

**Comment by eemhu** [comment]
Is there some additional configuration needed or is this a bug?

**Comment by kortemik** [comment]
it's a bug, I will check it out

**Comment by kortemik** [comment]
fixed in #14
